.github/workflows/push-processor.yaml

```diff
@@ -0,0 +1,35 @@
+---
+jobs:
+  build-push:
+    runs-on: 'ubuntu-latest'
+    steps:
+    - uses: 'actions/checkout@v4'
+    - id: 'metadata'
+      uses: 'docker/metadata-action@v5'
+      with:
+        images: 'econialabs/emojicoin-dot-fun-indexer-processor'
+        tags: |
+          type=match,pattern=processor-v(.*),group=1
+    - uses: 'docker/setup-qemu-action@v3'
+    - uses: 'docker/setup-buildx-action@v3'
+    - uses: 'docker/login-action@v3'
+      with:
+        password: '${{ secrets.DOCKERHUB_TOKEN }}'
+        username: '${{ secrets.DOCKERHUB_USERNAME }}'
+    - uses: 'docker/build-push-action@v6'
+      with:
+        cache-from: 'type=gha'
+        cache-to: 'type=gha,mode=max'
+        context: 'rust'
+        file: 'rust/Dockerfile'
+        labels: '${{ steps.metadata.outputs.labels }}'
+        platforms: '${{ vars.DOCKER_IMAGE_PLATFORMS }}'
+        push: 'true'
+        tags: '${{ steps.metadata.outputs.tags }}'
+    timeout-minutes: 360
+name: 'Build processor Docker image and push to Docker Hub'
+'on':
+  push:
+    tags:
+    - 'processor-v*'
```

Comment:
- [ ] Since these will be tags across our entire fork, how about  `emojicoin-processor-v*`?
- [ ] And, for consistent versioning between crate and docker image, how about bumping the processor version in Cargo.toml to one more than whatever is on Docker Hub currently? e.g. `0.x.0`